prog/test/ha_postgres_resource.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+require_relative "../../lib/util"
+
+class Prog::Test::HaPostgresResource < Prog::Test::Base
+  semaphore :destroy
+
+  def self.assemble
+    postgres_service_project = Project.create(name: "Postgres-Service-Project") { _1.id = Config.postgres_service_project_id }
```

Comment:
`postgres_standard` test also creates PG service project. It probably fails due to duplicate id.